pkg/deployments/README.md

```diff
@@ -101,7 +101,6 @@ Returns an object with all contracts from a deployment and their addresses.
 | Linear Pools for Aave aTokens (with rebalancing) V4    | [`20230206-aave-rebalanced-linear-pool-v4`](./tasks/20230206-aave-rebalanced-linear-pool-v4)         |
 | Linear Pools for Euler Tokens                          | [`20230208-euler-linear-pool`](./tasks/20230208-euler-linear-pool)                                   |
 | Linear Pools for Yearn Tokens                          | [`20230213-yearn-linear-pool`](./tasks/20230213-yearn-linear-pool)                                   |
-| Linear Pools for Gearbox Tokens                        | [`20230213-gearbox-linear-pool`](./tasks/20230213-gearbox-linear-pool)                               |
```

Comment:
Needs active entry for 20230409-gearbox-linear-pool-v2

Reply:
Done!